docs/2-how-it-works/momento-concepts.mdx

```diff
@@ -121,25 +121,19 @@ As noted above, Momento cache is a remote service from your application, and thu
 
 #### Default networking (public internet + HTTPS)
 
-By default, your SimpleCache client will connect to a publicly accessible endpoint for the Momento cache service. All requests to the Momento service will travel over the public internet to a Momento-managed load balancer, which will route your request to the proper server.
+By default, your SimpleCache client will connect to a publicly accessible endpoint for the Momento cache service. There are two main benefits to the default networking setup.
 
-![Architecture diagram of application interacting with Momento via public internet](images/momento-concepts/momento-public.png "Momento public access")
-
-There are two benefits to the default networking setup.
-
-First, it is simple. You can [get started with Momento](./../getting-started) and start interacting with your cache in as little as five minutes. You don't have to mess with complicated network configuration to benefit from a cache in your application.
+First, it is simple. You can [get started with Momento](./../getting-started) and start interacting with your cache in as little as five minutes right from your laptop. You don't have to mess with complicated network configuration to benefit from a cache in your application.
 
 Second, it works well with serverless applications. Many serverless applications prioritize using services that are available over the public internet using HTTPS. Part of this is due to the initial complexity and latency around using Lambda functions within a VPC, but part of it is due to the reduced complexity of avoiding complex network configurations.
```

Comment:
How about:

`Many serverless applications prioritize using services that are available over the public internet using HTTPS.` --> `Many serverless applications prioritize using services that are available using HTTPS rather than connection models that require a private VPC to control access.`